src/test/java/org/broadinstitute/hellbender/tools/copynumber/DetermineGermlineContigPloidyIntegrationTest.java

```diff
@@ -37,6 +38,38 @@ public void testCohort() {
         runCommandLine(argsBuilder);
     }
 
+    @Test
```

Comment:
There are some special things with the `@Test` annotation that can handle some of this.  

Something like this:

`@Test(expectedExceptions="RuntimeException.class", expectedExceptionsMessageRegExp="Tools requiring Python cannot be run in the Gatk Lite docker image.")`

Reply:
In general I would try to define a more specific exception type than runtime, some highly specific user exception usually and look for that instead of trying to scrape the message output.